File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        size = len(nums)
        
        if (size<2):
            return size
        
        slow,fast = 1,1
        
        while(fast < size):
            
            if(nums[slow-1] != nums[fast]):
                nums[slow] = nums[fast]
                
                #print(nums[slow],slow , nums)
                slow +=1
            
            fast +=1
        
        #print(slow , nums)
        return (slow)

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        
        match nums.is_empty(){
            true => 0,
            false => {
                let mut prev = 0;
                
                for index in 1..nums.len(){
                    if (nums[index] != nums[prev]){
                        prev +=1;
                        nums[prev] = nums[index];
                    }
                }
                (prev+1) as i32
            }
            
        }
    }
}